diff -ur cufflinks-1.0.1/src/abundances.h cufflinks-1.0.1.new/src/abundances.h
--- cufflinks-1.0.1/src/abundances.h	2011-05-04 21:17:15.000000000 +0100
+++ cufflinks-1.0.1.new/src/abundances.h	2011-05-17 11:50:34.796140381 +0100
@@ -15,8 +15,8 @@
 
 #include <stdint.h>
 #include <vector>
-#include <boost/numeric/ublas/matrix.hpp>
-#include <boost/numeric/ublas/vector.hpp>
+#include <boost141/boost/numeric/ublas/matrix.hpp>
+#include <boost141/boost/numeric/ublas/vector.hpp>
 #include "hits.h"
 #include "scaffolds.h"
 #include "bundles.h"
diff -ur cufflinks-1.0.1/src/biascorrection.h cufflinks-1.0.1.new/src/biascorrection.h
--- cufflinks-1.0.1/src/biascorrection.h	2011-05-06 20:54:26.000000000 +0100
+++ cufflinks-1.0.1.new/src/biascorrection.h	2011-05-17 11:51:22.397591223 +0100
@@ -11,12 +11,12 @@
  */
 
 
-#include <boost/numeric/ublas/matrix.hpp>
+#include <boost141/boost/numeric/ublas/matrix.hpp>
 #include <vector>
 #include <list>
 #include <string>
-#include <boost/tr1/unordered_map.hpp>
-#include <boost/thread.hpp>
+#include <boost141/boost/tr1/unordered_map.hpp>
+#include <boost141/boost/thread.hpp>
 #include "common.h"
 
 class MateHit;
diff -ur cufflinks-1.0.1/src/bundles.h cufflinks-1.0.1.new/src/bundles.h
--- cufflinks-1.0.1/src/bundles.h	2011-05-04 21:17:30.000000000 +0100
+++ cufflinks-1.0.1.new/src/bundles.h	2011-05-17 11:51:37.414417933 +0100
@@ -12,7 +12,7 @@
 #ifdef HAVE_CONFIG_H
 #include <config.h>
 #endif
-#include <boost/bind.hpp>
+#include <boost141/boost/bind.hpp>
 #include <vector>
 #include <numeric>
 #include "common.h"
diff -ur cufflinks-1.0.1/src/clustering.h cufflinks-1.0.1.new/src/clustering.h
--- cufflinks-1.0.1/src/clustering.h	2010-10-11 20:25:27.000000000 +0100
+++ cufflinks-1.0.1.new/src/clustering.h	2011-05-17 11:53:11.559331028 +0100
@@ -14,19 +14,19 @@
 #endif
 
 // DON'T move this, or mystery compiler errors will result. Affects gcc >= 4.1
-#include <boost/graph/vector_as_graph.hpp>
+#include <boost141/boost/graph/vector_as_graph.hpp>
 
-#include <boost/graph/adjacency_list.hpp>
-#include <boost/graph/depth_first_search.hpp>
-#include <boost/graph/visitors.hpp>
-#include <boost/graph/graph_traits.hpp>
-#include <boost/graph/connected_components.hpp>
+#include <boost141/boost/graph/adjacency_list.hpp>
+#include <boost141/boost/graph/depth_first_search.hpp>
+#include <boost141/boost/graph/visitors.hpp>
+#include <boost141/boost/graph/graph_traits.hpp>
+#include <boost141/boost/graph/connected_components.hpp>
 
 #ifdef DEBUG
-#include <boost/numeric/ublas/io.hpp>
+#include <boost141/boost/numeric/ublas/io.hpp>
 #endif
 
-#include <boost/version.hpp>
+#include <boost141/boost/version.hpp>
 
 #if (BOOST_VERSION < 103800)
 #include <boost/vector_property_map.hpp>
diff -ur cufflinks-1.0.1/src/common.h cufflinks-1.0.1.new/src/common.h
--- cufflinks-1.0.1/src/common.h	2011-05-04 21:17:39.000000000 +0100
+++ cufflinks-1.0.1.new/src/common.h	2011-05-17 11:53:44.288952963 +0100
@@ -18,15 +18,15 @@
 #include <string>
 #include <utility>
 
-#include <boost/math/distributions/normal.hpp> 
+#include <boost141/boost/math/distributions/normal.hpp> 
 using boost::math::normal;
 
-#include <boost/foreach.hpp>
+#include <boost141/boost/foreach.hpp>
 #define foreach         BOOST_FOREACH
 #define reverse_foreach BOOST_REVERSE_FOREACH
 
-#include <boost/thread.hpp>
-#include <boost/shared_ptr.hpp>
+#include <boost141/boost/thread.hpp>
+#include <boost141/boost/shared_ptr.hpp>
 
 // Non-option globals
 extern bool final_est_run;
diff -ur cufflinks-1.0.1/src/differential.h cufflinks-1.0.1.new/src/differential.h
--- cufflinks-1.0.1/src/differential.h	2011-05-04 21:17:36.000000000 +0100
+++ cufflinks-1.0.1.new/src/differential.h	2011-05-17 11:54:01.297756451 +0100
@@ -20,9 +20,9 @@
 #include <vector>
 #include <string>
 
-#include <boost/thread.hpp>
-#include <boost/graph/adjacency_list.hpp>
-#include <boost/graph/graph_traits.hpp>
+#include <boost141/boost/thread.hpp>
+#include <boost141/boost/graph/adjacency_list.hpp>
+#include <boost141/boost/graph/graph_traits.hpp>
 
 #include "abundances.h"
 #include "jensen_shannon.h"
diff -ur cufflinks-1.0.1/src/genes.cpp cufflinks-1.0.1.new/src/genes.cpp
--- cufflinks-1.0.1/src/genes.cpp	2011-05-04 21:17:12.000000000 +0100
+++ cufflinks-1.0.1.new/src/genes.cpp	2011-05-17 12:08:19.147562321 +0100
@@ -6,7 +6,7 @@
  *  Copyright 2009 Cole Trapnell. All rights reserved.
  *
  */
-#include <boost/thread.hpp>
+#include <boost141/boost/thread.hpp>
 #include "genes.h"
 
 using namespace boost;
diff -ur cufflinks-1.0.1/src/graph_optimize.h cufflinks-1.0.1.new/src/graph_optimize.h
--- cufflinks-1.0.1/src/graph_optimize.h	2010-10-11 20:25:27.000000000 +0100
+++ cufflinks-1.0.1.new/src/graph_optimize.h	2011-05-17 11:54:44.681255096 +0100
@@ -11,8 +11,8 @@
 
 #include <vector>
 
-#include <boost/graph/depth_first_search.hpp>
-#include <boost/graph/visitors.hpp>
+#include <boost141/boost/graph/depth_first_search.hpp>
+#include <boost141/boost/graph/visitors.hpp>
 
 #include "bundles.h"
 #include "scaffold_graph.h"
diff -ur cufflinks-1.0.1/src/hits.h cufflinks-1.0.1.new/src/hits.h
--- cufflinks-1.0.1/src/hits.h	2011-05-04 21:17:32.000000000 +0100
+++ cufflinks-1.0.1.new/src/hits.h	2011-05-17 11:55:15.996893091 +0100
@@ -14,7 +14,7 @@
 #include <vector>
 #include <cassert>
 
-#include <boost/shared_ptr.hpp>
+#include <boost141/boost/shared_ptr.hpp>
 
 #include <bam/sam.h>
 
diff -ur cufflinks-1.0.1/src/jensen_shannon.h cufflinks-1.0.1.new/src/jensen_shannon.h
--- cufflinks-1.0.1/src/jensen_shannon.h	2011-05-04 21:17:32.000000000 +0100
+++ cufflinks-1.0.1.new/src/jensen_shannon.h	2011-05-17 11:55:41.947586015 +0100
@@ -7,11 +7,11 @@
  *
  */
 
-#include <boost/numeric/ublas/matrix.hpp>
-#include <boost/numeric/ublas/matrix_proxy.hpp>
-#include <boost/numeric/ublas/vector.hpp>
-#include <boost/numeric/ublas/vector_proxy.hpp>
-#include <boost/numeric/ublas/io.hpp>
+#include <boost141/boost/numeric/ublas/matrix.hpp>
+#include <boost141/boost/numeric/ublas/matrix_proxy.hpp>
+#include <boost141/boost/numeric/ublas/vector.hpp>
+#include <boost141/boost/numeric/ublas/vector_proxy.hpp>
+#include <boost141/boost/numeric/ublas/io.hpp>
 
 #include <vector>
 
diff -ur cufflinks-1.0.1/src/multireads.h cufflinks-1.0.1.new/src/multireads.h
--- cufflinks-1.0.1/src/multireads.h	2011-05-04 21:17:37.000000000 +0100
+++ cufflinks-1.0.1.new/src/multireads.h	2011-05-17 11:55:56.971407008 +0100
@@ -1,7 +1,7 @@
 #ifndef MULTIREADS_H
 #define MULTIREADS_H
 
-#include <boost/thread.hpp>
+#include <boost141/boost/thread.hpp>
 
 typedef uint64_t RefID;
 typedef uint64_t InsertID;
diff -ur cufflinks-1.0.1/src/scaffold_graph.h cufflinks-1.0.1.new/src/scaffold_graph.h
--- cufflinks-1.0.1/src/scaffold_graph.h	2010-10-11 20:25:26.000000000 +0100
+++ cufflinks-1.0.1.new/src/scaffold_graph.h	2011-05-17 11:56:20.628125141 +0100
@@ -11,10 +11,10 @@
 
 #include <vector>
 #include <utility>
-#include <boost/graph/adjacency_list.hpp>
-#include <boost/graph/graph_traits.hpp>
+#include <boost141/boost/graph/adjacency_list.hpp>
+#include <boost141/boost/graph/graph_traits.hpp>
 
-#include <boost/version.hpp>
+#include <boost141/boost/version.hpp>
 
 #if (BOOST_VERSION < 103800)
 #include <boost/vector_property_map.hpp>
diff -ur cufflinks-1.0.1/src/scaffolds.h cufflinks-1.0.1.new/src/scaffolds.h
--- cufflinks-1.0.1/src/scaffolds.h	2011-05-04 21:17:16.000000000 +0100
+++ cufflinks-1.0.1.new/src/scaffolds.h	2011-05-17 11:56:30.421008455 +0100
@@ -19,7 +19,7 @@
 #include "common.h"
 #include "hits.h"
 
-#include <boost/thread.hpp>
+#include <boost141/boost/thread.hpp>
 
 using namespace std;
 
diff -ur cufflinks-1.0.1/src/transitive_closure.h cufflinks-1.0.1.new/src/transitive_closure.h
--- cufflinks-1.0.1/src/transitive_closure.h	2010-10-11 20:25:27.000000000 +0100
+++ cufflinks-1.0.1.new/src/transitive_closure.h	2011-05-17 11:57:17.829443569 +0100
@@ -26,13 +26,13 @@
 #include <vector>
 #include <algorithm> // for std::min and std::max
 #include <functional>
-#include <boost/config.hpp>
-#include <boost/bind.hpp>
-#include <boost/graph/vector_as_graph.hpp>
-#include <boost/graph/strong_components.hpp>
-#include <boost/graph/topological_sort.hpp>
-#include <boost/graph/graph_concepts.hpp>
-#include <boost/graph/named_function_params.hpp>
+#include <boost141/boost/config.hpp>
+#include <boost141/boost/bind.hpp>
+#include <boost141/boost/graph/vector_as_graph.hpp>
+#include <boost141/boost/graph/strong_components.hpp>
+#include <boost141/boost/graph/topological_sort.hpp>
+#include <boost141/boost/graph/graph_concepts.hpp>
+#include <boost141/boost/graph/named_function_params.hpp>
 	
 using namespace boost;
 
diff -ur cufflinks-1.0.1/src/transitive_reduction.h cufflinks-1.0.1.new/src/transitive_reduction.h
--- cufflinks-1.0.1/src/transitive_reduction.h	2010-10-11 20:25:26.000000000 +0100
+++ cufflinks-1.0.1.new/src/transitive_reduction.h	2011-05-17 11:57:51.215045761 +0100
@@ -9,11 +9,11 @@
 
 #include <vector>
 #include <algorithm> //std::find
-#include <boost/concept/requires.hpp>
-#include <boost/concept_check.hpp>
+#include <boost141/boost/concept/requires.hpp>
+#include <boost141/boost/concept_check.hpp>
 
-#include <boost/graph/graph_traits.hpp>
-#include <boost/graph/topological_sort.hpp>
+#include <boost141/boost/graph/graph_traits.hpp>
+#include <boost141/boost/graph/topological_sort.hpp>
 
 // also I didn't got all of the concepts thin. Am I suppose to check
 // for all concepts, which are needed for functions I call? (As if I
