diff -ur tophat-1.3.0/ax_bam.m4 tophat-1.3.0.new/ax_bam.m4
--- tophat-1.3.0/ax_bam.m4	2010-10-11 20:24:42.000000000 +0100
+++ tophat-1.3.0.new/ax_bam.m4	2011-06-06 12:34:02.313547400 +0100
@@ -102,7 +102,7 @@
 
 	AC_LANG_PUSH(C++)
      	AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[
-	@%:@include <bam/bam.h>
+	@%:@include <samtools/bam.h>
 	]], [[
 	]])],[
         AC_MSG_RESULT(yes)
diff -ur tophat-1.3.0/src/align_status.h tophat-1.3.0.new/src/align_status.h
--- tophat-1.3.0/src/align_status.h	2010-12-09 20:25:09.000000000 +0000
+++ tophat-1.3.0.new/src/align_status.h	2011-06-06 12:34:02.313547400 +0100
@@ -14,7 +14,7 @@
 #include <cassert>
 #include <cstring>
 #include <seqan/sequence.h>
-#include <bam/sam.h>
+#include <samtools/sam.h>
 #include "common.h"
 
 
diff -ur tophat-1.3.0/src/bwt_map.h tophat-1.3.0.new/src/bwt_map.h
--- tophat-1.3.0/src/bwt_map.h	2011-06-02 22:56:55.000000000 +0100
+++ tophat-1.3.0.new/src/bwt_map.h	2011-06-06 12:34:02.313547400 +0100
@@ -15,7 +15,7 @@
 #include <cstring>
 #include <seqan/sequence.h>
 
-#include <bam/sam.h>
+#include <samtools/sam.h>
 using namespace std;
 
 #include "common.h"
diff -ur tophat-1.3.0/src/common.h tophat-1.3.0.new/src/common.h
--- tophat-1.3.0/src/common.h	2011-06-02 22:56:56.000000000 +0100
+++ tophat-1.3.0.new/src/common.h	2011-06-06 12:33:36.614841952 +0100
@@ -15,8 +15,8 @@
 #include <cstdio>
 #include <string>
 #include <vector>
-#include "bam/bam.h"
-#include "bam/sam.h"
+#include "samtools/bam.h"
+#include "samtools/sam.h"
 
 
 #ifdef MEM_DEBUG
